### ISSUE TITLE
Error: rustdoc output format is too new (v19, need v18) when parsing /tmp/baseline.json

### ISSUE THREAD

**Issue by scouten-adobe** [obi1kenobi/cargo-semver-check#122]
### Steps to reproduce the bug with the above code

Run cargo-semver-check-action against a PR.

It appears the nightly build of Rustdoc has a new output format which is not supported by this crate.

### Actual Behaviour

Excerpted from https://github.com/contentauth/c2pa-rs/runs/8216717251?check_suite_focus=true:

```
+ cargo semver-checks check-release --current /tmp/current.json --baseline /tmp/baseline.json
Error: rustdoc output format is too new (v19, need v18) when parsing /tmp/baseline.json

note: a newer version of cargo-semver-checks is likely available
```

### Expected Behaviour

No such error occurs.

I checked and this installed 0.9.2 of the crate, which is the latest as of this writing.

**Comment by obi1kenobi** [comment]
Apologies, this is my mistake — I set up a nightly test against the most recent Rust nightly to try to catch this issue, but our test suite wasn't actually guaranteed to spot the problem in cases where the format break was relatively tiny, as it was in this case. This is why our nightly builds were all green even while the problem existed.

I've added a direct "does the format match what we support" test case so we as maintainers can spot the format break sooner next time, ideally before anyone else in the community notices. Longer term, we obviously want something even more robust here.

v0.10.0 of `cargo-semver-checks` is building right now and should be published in ~10min with the fix.
